### PR TITLE
MRKT-10 Add Success/Unsuccessful page for token

### PR DESCRIPTION
**Pull request by robdev91**
Ticket Link
[MRKT-10](https://mojitoinc.atlassian.net/browse/MRKT-10)

### CODE REVIEW

File: src/components/item/index.ts [hunk 1]
```diff
@@ -7,3 +7,4 @@ export * from "./BuyNowDetail";
 export * from "./BuyNowModal";
 export * from "./ItemComponents";
 export * from "./ModalComponents";
+export * from "./TokenDetail";
```
Comment (Dmckinney821) [inline src/components/item/index.ts:10]: We need an extra line here

File: src/pages/purchase/[id].tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+const getServerSideProps: GetServerSideProps = async (context) => {
+  const mojitoId = context.params?.id;
+
+  if (typeof mojitoId !== "string") {
+    return getRedirect();
+  }
+
+  const token = convertToToken(mojitoId);
+  
+  if (hasAccessToToken(token)) {
```
Comment (Dmckinney821) [inline src/pages/purchase/[id].tsx:38]: Would ternary be better here?

File: src/pages/purchase/[id].tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import type { GetServerSideProps, NextPage } from "next";
```
Comment (Dmckinney821) [inline src/pages/purchase/[id].tsx:1]: Is this replacing `[token].tsx` file that was deleted below?

File: src/components/item/index.ts (hunk 1 above)
Comment (robdev91) [reply src/components/item/index.ts]: Added.

File: src/pages/purchase/[id].tsx (hunk 2 above)
Comment (robdev91) [reply src/pages/purchase/[id].tsx]: Changed into ternary and looks better. :)

File: src/pages/purchase/[id].tsx (hunk 3 above)
Comment (robdev91) [reply src/pages/purchase/[id].tsx:1]: Correct.

**Review by Dmckinney821 — APPROVED** [review]

**Pull request merged**